media_driver/agnostic/common/cm/cm_mem_sse2_impl.cpp

```diff
@@ -39,7 +39,7 @@ void FastMemCopy_SSE2_movntdq_movdqa(
     CM_ASSERT( IsAligned( dst, sizeof(DQWORD) ) );
     CM_ASSERT( IsAligned( src, sizeof(DQWORD) ) );
 
-    const size_t doubleQuadWordsPerPrefetch = sizeof(PREFETCH) / sizeof(DQWORD);
+    const size_t doubleQuadWordsPerPrefetch = sizeof(PREFETCH) / (sizeof(DQWORD));
```

Comment:
To code owners: is that possible to redefine PREFETCH to DQWORD to avoid the need to additional `()`?

Reply:
Maybe no. PREFETCH is a 32-byte type, and its size is double of DQWORD.

Reply:
Thank you, Fan. So, this needs to remain as is. I updated PR follow suggestions from @eero-t. Introduced a macro and added a comment why we make this change.